feat(i18n): add Simplified Chinese

xcfox opened the pull request:
add Simplified Chinese for `@valibot/i18n`

fabian-hiller commented:
Thank you very much for this contribution! Do you know another native speaker who could review your translations for me?

fabian-hiller commented:
Can you explain the difference between Simplified Chinese (zh-CN) and Chinese (zh)?

xcfox commented:
> Can you explain the difference between Simplified Chinese (zh-CN) and Chinese (zh)?

Modern Chinese is divided into Simplified Chinese and Traditional Chinese. Simplified Chinese and Traditional Chinese are two ways of writing the same Chinese character. Simplified Chinese is popular in Mainland China, Singapore and Malaysia, while Traditional Chinese is popular in Taiwan Province of China and Hong Kong SAR of China.

现代中文分为简体中文和繁体中文。简体和繁体是同一个汉字的两种写法。简体中文在中国大陆、新加坡、马来西亚流行，繁体中文在中国台湾省、中国香港流行。

現代中文分為簡體中文和繁體中文。簡體和繁體是同一個漢字的兩種寫法。簡體中文在中國大陸、新加坡、馬來西亞流行，繁體中文在中國臺灣省、中國香港流行。

0712uu commented on packages/i18n/src/zh-CN.ts:
```diff
@@ -0,0 +1,66 @@
+    endsWith:       (issue) => `无效的结尾：预期结尾为 ${issue.expected}，而输入为 ${issue.received}`,
+    equal:          (issue) => `无效的值：预期为 ${issue.expected}，而输入为 ${issue.received}`,
+    excludes:       (issue) => `无效的内容：预期为 ${issue.expected}，而输入为 ${issue.received}`,
+    finite:         (issue) => `无效的无限值：输入为 ${issue.received}`,
```
这里应该是“有限的”

0712uu reviewed (changes requested):
I am a native speaker of Chinese. I have made some review.

xcfox commented:
The misstake is fixed

fabian-hiller commented:
I will review it in the next days!

fabian-hiller commented:
Could one or both of you also review the `zh-TW` translations in #427?

Mini-ghost commented:
In fact, Traditional Chinese and Simplified Chinese not only have different writing methods, but they also feature distinct idiomatic expressions.

After reading the Simplified Chinese translation, I believe it is great. Additionally, the PR has been reviewed by Simplified Chinese users, and I believe the results are trustworthy.

fabian-hiller commented:
Thank you for your review!
